src/main/resources/metafile-schema.json

```diff
@@ -48,6 +48,12 @@
       "title": "The name of the zip file",
       "pattern": "^\\d+_([012][0-9]|30|31)-([0][0-9]|[1][012])-[2][0][0-9][0-9]-([01][0-9]|[2][0123])-[0-5][0-9]-[0-5][0-9]\\.(test\\.)?zip$"
     },
+    "rescan_for": {
+      "id": "#/properties/rescan_for",
+      "type": "string",
```

Comment:
can it be null?

Reply:
no, I'll update the `type` values to support null